test/TestConfig/settings.mk

```diff
@@ -212,9 +212,11 @@ ifndef UNIQUEID
 endif
 TESTOUTPUT := $(TEST_ROOT)$(D)TestConfig$(D)test_output_$(UNIQUEID)
 ifeq ($(TEST_ITERATIONS), 1)
-	REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@$(Q)
+	REPORTDIR_NQ = $(TESTOUTPUT)$(D)$@
+	REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)
 else
-	REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@_ITER_$$itercnt$(Q)
+	REPORTDIR_NQ = $(TESTOUTPUT)$(D)$@_ITER_$$itercnt
+	REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)
```

Comment:
Should we set `REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)` after if else?

Reply:
fixed